src/components/Button.js

```diff
@@ -0,0 +1,40 @@
+import './styles.css';
+
+function Button() {
+  return (
+    <div className="operations subgrid">
+
+      {/* division btn */}
+      <button type="button">
+        {' '}
+        ÷
+      </button>
+
+      {/* multiplication btn */}
+      <button type="button">
+        {' '}
+        x
+      </button>
+
+      {/* minus btn */}
+      <button type="button">
+        {' '}
+        -
+        {' '}
+      </button>
+
+      {/* add button */}
+      <button type="button">
+        +
+      </button>
+
+      {/* "=" btn */}
+      <button
+        type="button"
+      >
+        =
+      </button>
+    </div>
+  );
+}
+export default Button;
```

Comment:
- [ ] Great job creating a `child component` as suggested by the previous reviewer. However, I can see you're not making use of this child component anywhere in your project which is more like you not creating it at all. Kindly make use of the `child component` you have created.